cfme/configure/settings.py

```diff
@@ -396,6 +401,24 @@ def get_default_view(cls, button_group_name, fieldset=None):
         navigate_to(cls, 'All')
         return bg.active
 
+    @classmethod
+    def set_default_view_switch_on(cls):
+        navigate_to(cls, 'All')
+        appliance = get_or_create_current_appliance()
+        views = appliance.browser.widgetastic.create_view(DefaultBootstrapSwitch)
+        if views.bs.fill(True):
+            sel.click(form_buttons.save)
```

Comment:
I'm afraid you have to rewrite whole form to widgetastic. `sel.click` must go. It seems your changes requires more work than expected.